lnwire/accept_channel.go

```diff
@@ -126,23 +126,63 @@ func (a *AcceptChannel) Encode(w *bytes.Buffer, pver uint32) error {
 		return err
 	}
 
-	return WriteElements(w,
-		a.PendingChannelID[:],
-		a.DustLimit,
-		a.MaxValueInFlight,
-		a.ChannelReserve,
-		a.HtlcMinimum,
-		a.MinAcceptDepth,
-		a.CsvDelay,
-		a.MaxAcceptedHTLCs,
-		a.FundingKey,
-		a.RevocationPoint,
-		a.PaymentPoint,
-		a.DelayedPaymentPoint,
-		a.HtlcPoint,
-		a.FirstCommitmentPoint,
-		tlvRecords,
-	)
+	if err := WriteBytes(w, a.PendingChannelID[:]); err != nil {
```

Comment:
Extra `I` in the commit message?

Reply:
Ahh it's meant to be like "part 1" gotcha.

Reply:
😂

Reply:
Actually it's missing a dash...will put it back